README.md

```diff
@@ -2,6 +2,14 @@
 
 This is a terraform mixin for [Porter](https://github.com/deislabs/porter).
 
+## Install via Porter
+
+This will install the latest mixin release via the Porter CLI.
+
+```console
```

Comment:
markdown tip: `console` is good for when you want to show a command (prefixed with `$`) followed on the next line by the output. The command will be formatted in black and the output in blue.

When I am just showing a command all by itself, ready to copy/paste when the user triple clicks, I usually omit the `$` and not specify the code fence type so that it shows up in black. This is a style preference but once that I've been sticking to in the porter docs.

Reply:
Ah, thank you!  `console` type removed to better reflect the intended application (copy/paste).